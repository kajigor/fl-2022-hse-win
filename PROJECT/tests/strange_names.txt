<S> = '\E' ;
<S> = '\'';
<S> = <\<\>>;
<\<\>> = '\\';
<S> = 'E';
