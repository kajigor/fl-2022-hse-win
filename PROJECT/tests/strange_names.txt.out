LexToken(START_NON_TERM,'S',1,0)
LexToken(EQ,'=',1,4)
LexToken(TERM,'E',1,6)
LexToken(END,';',1,11)
LexToken(START_NON_TERM,'S',2,0)
LexToken(EQ,'=',2,4)
LexToken(TERM,"'",2,6)
LexToken(END,';',2,10)
LexToken(START_NON_TERM,'S',3,0)
LexToken(EQ,'=',3,4)
LexToken(NON_TERM,'<>',3,6)
LexToken(END,';',3,12)
LexToken(NON_TERM,'<>',4,0)
LexToken(EQ,'=',4,7)
LexToken(TERM,'\\',4,9)
LexToken(END,';',4,13)
LexToken(START_NON_TERM,'S',5,0)
LexToken(EQ,'=',5,4)
LexToken(EPSILON,'E',5,6)
LexToken(END,';',5,9)
